Gemfile.lock

```diff
@@ -1,16 +1,9 @@
-GIT
-  remote: https://github.com/puppetlabs/wash-ruby
-  revision: ab5d9b8504c90152aeb46059c549d3a86ccee6da
+PATH
```

Comment:
Hmm, I never had to include Gemfile.lock files before? What's the reason for including it?

Reply:
We don't really need to, but it was already there. It makes what's "deployed" deterministic (you'll use exactly the versions in the lock file).

Reply:
Ok, that makes sense.